src/Summoner/Template.hs

```diff
@@ -326,31 +331,26 @@ createProjectTemplate ProjectData{..} = Dir (toString repo) $
             "http://stackage.org/package/" <> repo <> "/badge/nightly"
         stackLinkNightly :: Text =
             "http://stackage.org/nightly/package/" <> repo
-
-        stackBadges :: Text = memptyIfFalse stack
-            [text|
-            [![Stackage Lts](${stackShieldLts})](${stackLinkLts})
-            [![Stackage Nightly](${stackShieldNightly})](${stackLinkNightly})
-            |]
-
+        stackBadges :: Text 
+            = makeBadge "Stackage Lts" stackShieldLts stackLinkLts
+           <> makeBadge "Stackage Nightly" stackShieldNightly stackLinkNightly
+            
         travisShield :: Text =
-          "https://secure.travis-ci.org/" <> owner <> "/" <> repo <> ".svg"
+            "https://secure.travis-ci.org/" <> owner <> "/" <> repo <> ".svg"
         travisLink :: Text =
-          "https://travis-ci.org/" <> owner <> "/" <> repo
-        travisBadge :: Text = memptyIfFalse travis
-            [text|[![Build status](${travisShield})](${travisLink})|]
+            "https://travis-ci.org/" <> owner <> "/" <> repo
+        travisBadge :: Text = 
+            makeBadge "Build status" travisShield travisLink
 
         appVeyorShield :: Text =
-          "https://ci.appveyor.com/api/projects/status/github/" <> owner <> "/" <> repo <> "?branch=master&svg=true"
+            "https://ci.appveyor.com/api/projects/status/github/" <> owner <> "/" <> repo <> "?branch=master&svg=true"
         appVeyorLink :: Text =
-          "https://ci.appveyor.com/project/" <> owner <> "/" <> repo
-        appVeyorBadge :: Text = memptyIfFalse appVey
-            [text|[![Windows build status](${appVeyorShield})](${appVeyorLink})|]
+            "https://ci.appveyor.com/project/" <> owner <> "/" <> repo
+        appVeyorBadge :: Text = 
+            makeBadge "Windows build status" appVeyorShield appVeyorLink
 
-        licenseShield :: Text =
-          "https://img.shields.io/badge/license-" <> T.replace "-" "--" license <> "-blue.svg"
-        licenseLink :: Text =
-          "https://github.com/" <> owner <> "/" <> repo <> "/blob/master/LICENSE"
+        makeBadge :: Text -> Text -> Text -> Text
+        makeBadge title shield link = "[![" <> title <> "](" <> shield <> ")](" <> link <> ")"
```

Comment:
If you like, you could use `neat-interpolation` in `makeBadge` function:

```haskell
[text|[![$title]($shield)]($link)|]
```

Choose which one you like more :slightly_smiling_face: I'm okay with both choices :+1: